### ISSUE TITLE
Bug parse CPT as a result of a Polars PanicException

### ISSUE THREAD

**Issue by RDWimmers** [cemsbv/pygef#121]
Parsing the following cpt gives an error:

```
/opt/conda/lib/python3.7/site-packages/pygef/cpt.py in __init__(self, path, content)
    151             assert content["string"] is not None, "content['string'] must be specified"
    152             if content["file_type"] == "gef":
--> 153                 parsed = _GefCpt(string=content["string"])
    154             elif content["file_type"] == "xml":
    155                 parsed = _BroXmlCpt(string=content["string"])

/opt/conda/lib/python3.7/site-packages/pygef/gef.py in __init__(self, path, string)
    288                         calculate_friction_number(column_names),
    289                         self.calculate_elevation_with_respect_to_nap(
--> 290                             self.zid, self.height_system
    291                         ),
    292                     ]

/opt/conda/lib/python3.7/site-packages/polars/lazy/frame.py in collect(self, type_coercion, predicate_pushdown, projection_pushdown, simplify_expression, string_cache, no_optimization)
    297             string_cache,
    298         )
--> 299         return pl.eager.frame.wrap_df(ldf.collect())
    300 
    301     def fetch(

PanicException: python apply failed: Any(InvalidOperation("abs not supportedd for series of type Float64"))
```

[file.zip](https://github.com/cemsbv/pygef/files/7547248/file.zip)

How to reproduce:
`pygef.Cpt(path="./file.gef")`

**Comment by ritchie46** [comment]
Already known. Downgrade to `polars==0.10.19` for the time being. I will patch this in a few days.

**Comment by ritchie46** [comment]
And fixed in `polars==0.10.21`
